src/term/parser/parser.rs

```diff
@@ -260,10 +260,19 @@ where
     let str = select!(Token::Str(s) => Term::Str { val: s }).boxed();
     let chr = select!(Token::Char(c) => Term::Num { val: c }).boxed();
 
+    let list = term
+      .clone()
+      .separated_by(just(Token::Comma))
```

Comment:
We can make this comma optional, since it's just a separator and terms are self delimiting